qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -162,7 +162,7 @@ class CreateCustomerView extends FormLayout {
 
         this.lastNameField.addValueChangeListener({ event ->
             ValidationResult result = new StringLengthValidator("Please input a valid last name",
-                1, null).apply(event.getValue(), new ValueContext(view.lastNameField))
+                1, null).apply(event.getValue(), new ValueContext(this.lastNameField))
```

Comment:
```suggestion
                1, null).apply(event.getValue(), new ValueContext(lastNameField))
```
I do not know if `this` inside of the closure works since it technically is withing the Listener that is described by the closure.

Reply:
But I am not sure about this ... *cough* pun intended